src/client/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -91,8 +91,11 @@ public async Task<IEnumerable<IAccount>> GetAccountsAsync()
             if (AppConfig.IsBrokerEnabled && ServiceBundle.PlatformProxy.CanBrokerSupportSilentAuth())
             {
                 var broker = ServiceBundle.PlatformProxy.CreateBroker(null);
-                accounts = await broker.GetAccountsAsync(AppConfig.ClientId).ConfigureAwait(false);
-                return accounts;
+                if (broker.CanInvokeBroker())
```

Comment:
Checking to see if it is possible to invoke broker at all. if not, this will now continue on to the normal getAccountsAsync functionality W/O broker